.gitignore

```diff
@@ -37,6 +37,7 @@ package-lock.json
 !.idea/icon.svg
 **/.vscode/
 **/.fleet/
+compile_commands.json
```

Comment:
why add this?

Reply:
i will soft link compile_commands.json from build dir to project root dir.  I will discard this change.